src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerSchemaDataStore.cs

```diff
@@ -59,6 +59,84 @@ public async Task<GetCompatibilityVersionResponse> GetLatestCompatibleVersionAsy
             }
         }
 
+        public async Task<string> InsertInstanceSchemaInformation(string name, SchemaInformation schemaInformation, CancellationToken cancellationToken)
+        {
+            using (SqlConnectionWrapper sqlConnectionWrapper = _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapper())
+            using (SqlCommand sqlCommand = sqlConnectionWrapper.CreateSqlCommand())
+            {
+                VLatest.CreateInstanceSchema.PopulateCommand(
+                     sqlCommand,
+                     name,
+                     schemaInformation.Current.GetValueOrDefault(),
+                     schemaInformation.MaximumSupportedVersion,
+                     schemaInformation.MinimumSupportedVersion);
+                try
+                {
+                    var output = (string)await sqlCommand.ExecuteScalarAsync();
+
+                    if (output.NullIfEmpty() == null)
+                    {
+                        throw new OperationFailedException(Resources.OperationFailed);
+                    }
+
+                    return output;
```

Comment:
Do we need to return the `output`. Doesn't seem to be used by the caller.

Reply:
Good point, Addressed!